cmd/ltops/deploy.go

```diff
@@ -63,5 +79,7 @@ func init() {
 
 	deploy.Flags().StringP("loadtests", "t", "", "the loadtests package to use (required for terraform)")
 
+	deploy.Flags().IntP("users", "u", 0, "the number of active users to configure the load test to run with (required for kubernetes)")
```

Comment:
nit: long help text wraps awkwardly. Would the following suffice?
> the number of active users during the load test (required for kubernetes)

Reply:
I can shorten it 👍